src/Symfony/Component/Mailer/Bridge/MicrosoftGraph/composer.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "symfony/microsoft-graph-mailer",
+    "type": "symfony-mailer-bridge",
+    "description": "Symfony Microsoft Graph Mailer Bridge",
+    "keywords": [],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Bob van de Vijver",
+            "homepage": "https://github.com/bobvandevijver"
+        },
+        {
+            "name": "Kevin Nguyen",
+            "homepage": "https://github.com/nguyenk"
+        },
+        {
+            "name": "The Coding Machine",
+            "homepage": "https://github.com/thecodingmachine"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.2",
+        "symfony/clock": "^7.4|^8.0",
+        "symfony/mailer": "^7.4|^8.0"
+    },
+    "require-dev": {
+        "symfony/http-client": "^6.4|^7.0|^8.0"
```

Comment:
Should be moved to the `require` section IMO if the HTTP-based API is the only way to use this bridge.